README.md

```diff
@@ -1 +1,265 @@
-# dapp-runner
+# Golem dApp Runner
+
+`dapp-runner` is a utility that allows you to run decentralized applications on [Golem](https://www.golem.network/).
+It uses simple application descriptors expressed in `yaml`, similar to those used by
+tools like `docker-compose`.
+
+`dapp-runner` runs alongside the [Golem daemon](https://github.com/golemfactory/yagna)
+and uses [yapapi](https://github.com/golemfactory/yapapi), Golem's Python high-level API
+to communicate with it. As opposed to using plain `yapapi` though, deployment of
+applications on Golem using `dapp-runner` requires no code and no experience in Python.
+
```

Comment:
Would it be suitable to add a mention and reference to `dapp-manager`?

Reply:
sure, the dependency goes the other way round but I think there'd be no harm

Reply:
added